verity/src/helm/values.yaml

```diff
@@ -20,16 +25,48 @@ deployment:
       periodSeconds: 5
       timeoutSeconds: 2
       failureThreshold: 2
+  resources: # TODO tune the values
+    requests:
+      memory: 768Mi
+      cpu: 250m
+    limits:
+      memory: 1024Mi
+      cpu: 1000m
+  useAffinity: false # TODO requires better configuraton
+
 k8sService:
   type: NodePort
   port: 80
   targetPort: 9000
+
+ingress:
+  # if empty - internal ingress is used
+  targetGroupARNarn: ""
+  # internal ingress configuration
+  force_ssl: false
+  healthcheckPath: "/verity/node/readiness"
+  healthcheckIntervalSeconds: 5
+  healthcheckTimeoutSeconds: 2
+  unhealthyThresholdCount: 2
+  deregistrationDelay: 120
+
+# role arn to use for EKS service account
+secretsRoleArn: ""
+
+secrets:
+  # name/alias of the secret in AWS SecretsManager to put into credentials-tf.json
+  awsSecretsNameCredentialsTf: ""
+
 # List of sponsors for Verity
 sponsors:
 # List configuration for TAA.
 taa:
 # VDR configuration
 vdr:
+
+# application configuraion variables
+vars: {}
+
```

Comment:
why do we need this empty object assigned to `vars`?

Reply:
as we discussed it acts as a kind of docs/API for helm chart installers, default value as `{}` is good here